src/views/transaction/transaction.view.jsx

```diff
@@ -162,42 +162,27 @@ function Transaction ({
                           <Col>From</Col>
                           <Col link>
                             <Row wrapped>
-                              <CopyToClipboardButton content={transactionTask.data.fromAccountIndex} />
+                              <CopyToClipboardButton content={transactionTask.data.fromBjj} />
                               <Col wrapped>
                                 <Link to={`/user-account/${transactionTask.data.fromBjj}`}>
-                                  {transactionTask.data.fromAccountIndex}
+                                  {transactionTask.data.fromBjj}
```

Comment:
Could you also add this check https://github.com/hermeznetwork/wallet-ui/pull/416/files#diff-eb6529b939aabcee85cab3f33b6da1cace3a571d74ea8517411c8f59c33b3ce8R79 here and in the To section please? Basically to avoid: https://explorer.internaltestnet.hermez.io/transaction/0x027bb5c58a7ba5a32c03df8753e484dbb07edac120005ff246f898f457cae6ccc4

Reply:
Added.